### DISCUSSION TITLE
CMU 10-414 Assignments 实验笔记 | 周鑫的个人博客

### DISCUSSION THREAD

**Discussion by giscus** [LittleHeroZZZX/zhouxin-space#114] [Announcements]
# CMU 10-414 Assignments 实验笔记 | 周鑫的个人博客

前言 本文记录了完成《CMU 10-414/714 Deep Learning System》配套 Assignments 的过程和对应笔记。共有 6 个 hw，循序渐进地从头实现了一个深度学习框架，并利用搭建 DL 中厂常

https://www.zhouxin.space/notes/notes-on-cmu-10-414-assignments/

**Comment by kukutu** [comment]
求更新

**Reply by LittleHeroZZZX** [reply]
后面要写cuda代码了，没学过，想着先系统学一下cuda再接着做...

**Comment by Cpaulyz** [comment]
感谢文档～给了很大帮助！有个小问题：

https://www.zhouxin.space/notes/notes-on-cmu-10-414-assignments/#softmax_regression_epoch 

softmax_regression_epoch 第十行按照定义是不是应该 `/X_batch.shape[0]`？

**Reply by LittleHeroZZZX** [reply]
是的，这里没考虑到尾批中的size。原文中提到的双层感知机的精度错误想必也来自于此。感谢指出🙏
